sktime/registry/_tags.py

```diff
@@ -184,7 +184,7 @@
         "capability:pred_int",
         "forecaster",
         "bool",
-        "is the forecaster capable of returning prediction intervals in predict?",
+        "does the forecaster implement predict_interval and predict_quantiles?",
```

Comment:
```suggestion
        "does the forecaster implement predict_interval or predict_quantiles?",
```

Reply:
it's always "and" - if the one is implemented, the other one also works. So, no change needed?

Reply:
yes, but not both need to be implemented to require this tag to be `True`?

Reply:
yes, depending on how you interpret "implement". I guess the "or" version is less confusing, so changed to that.